doorman/models.py

```diff
@@ -387,3 +389,24 @@ class DistributedQueryResult(SurrogatePK, Model):
     def __init__(self, data, distributed_query=None):
         self.data = data
         self.distributed_query = distributed_query
+
+
+class Rule(SurrogatePK, Model):
+    __tablename__ = 'rule'
+
+    ADDED = 'added'
+    REMOVED = 'removed'
+    BOTH = 'both'
+
+    type = Column(db.String, nullable=False)
+    name = Column(db.String, nullable=False)
+    action = Column(db.Enum(ADDED, REMOVED, BOTH, name='rule_actions'), nullable=False)
+    alerters = Column(ARRAY(db.String), nullable=False)
```

Comment:
Yes, you can keep an empty array here - this just means it can't be `NULL` / `None`.